_posts/2020-12-01-getting-started-phoenix.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Primeiros passos com Phoenix
+author: Elixir Lab
+layout: post
+---
+# 1. Suposições desse guia
+Esse guia é feito para iniciantes. A gente não espera que você possua qualquer experiência com Phoenix (....)
+
+==talvez com um pouco de elixir.==
+
+# 2. O que é o Phoenix
+Phoenix é framework (==preciso explicar o que é framework?==) que nos auxilia na construção de applicações para web.
+(...)
+
+# 3. Criando seu primeiro projeto em Phoenix
+A melhor forma de aproveitar esse guia é seguir o passo a passo.
+
+Seguindo esse guia a gente vai terminar com um blog (simples, não muito bonito, mas um blog)
+
+mas antes de começar, precisamos garantir que você tem tudo instalado na sua máquina:
+
+* Erlang
+* Elixir
+* PostgreSQL
+* Node.JS
+* Um editor da sua preferencia (==trabalhar melhor aqui. talvez ja indicar o vscode?==)
+
+==Criar um guia de instalação para cada uma dessas ferramentas, no windows, linux e mac==
+
+Quando você tiver tudo instalado, estamos preparadas para instalar o phoenix.
+
+No terminal, digitar:
+
+==talvez deixar em algum lugar como faz pra lidar com o terminal? terminal pode ser assustador pra quem nunca teve contato com programação==
```

Comment:
Siim, acho super válido

No Joy of Elixir tem uma parte sobre terminal, vou adicionar como referência para gente depois